financial-connections/src/main/java/com/stripe/android/financialconnections/FinancialConnectionsSheetViewModel.kt

```diff
@@ -540,3 +545,7 @@ internal class FinancialConnectionsSheetViewModel @Inject constructor(
         return null
     }
 }
+
+private fun FinancialConnectionsSheetActivityArgs.retrieveElementsContext(): ElementsContext? {
+    return (this as? ForInstantDebits)?.elementsContext
+}
```

Comment:
this is ok for now but might break in the future if we add more integrations. 

Should we instead add it to the Args sealed class and pass null on non `InstantDebits` cases?

```
sealed class FinancialConnectionsSheetActivityArgs(
    open val configuration: FinancialConnectionsSheet.Configuration,
    **open val elementsContext: ElementsContext?**
) : Parcelable {

data class ForData(...) : FinancialConnectionsSheetActivityArgs(configuration, **null**)
```

Reply:
Good idea, added it for `FinancialConnectionsSheetActivityArgs`.